src/main/java/com/stripe/model/ExternalAccount.java

```diff
@@ -24,6 +24,7 @@
   String customer;
   @Getter(onMethod = @__({@Override})) Map<String, String> metadata;
 
+  // <editor-fold desc="delete">
```

Comment:
Because "external account" is really more of an implementation detail of the Java library and I wasn't sure how to surface it in the Javadoc :/

I'd kind of like to remove (or at least rework) external accounts in the next major version anyway.